### PR TITLE
Hive mobile menu

### PR DESCRIPTION
**Pull request by hasparus**
I'll point this to v7 when I stop using its snapshots.

### CODE REVIEW

File: packages/components/src/components/hive-navigation/index.tsx [hunk 1]
```diff
@@ -71,23 +69,46 @@ export function HiveNavigation({
   const themeConfig = useThemeConfig() as ReturnType<typeof useThemeConfig> | undefined;
   const Search = themeConfig?.search?.component;
 
+  const mobileMenu = useMenu();
+  console.log({ mobileMenu });
```
Comment (saihaj) [inline packages/components/src/components/hive-navigation/index.tsx:73]:
```suggestion
```

File: packages/components/src/components/hive-navigation/index.tsx [hunk 2]
```diff
@@ -71,23 +69,46 @@ export function HiveNavigation({
...
+
   const isHive = productName === 'Hive';
 
+  const logo = (
+    <Anchor href="/" className="-m-2 flex items-center rounded-md p-2">
+      <HiveCombinationMark className="text-green-1000 dark:text-neutral-200" />
+    </Anchor>
+  );
```
Comment (saihaj) [inline packages/components/src/components/hive-navigation/index.tsx:81]: you could just move this outside as a separate component given it doesn't change

File: packages/components/src/components/hive-navigation/index.tsx (hunk 1 above)
Comment (hasparus) [reply packages/components/src/components/hive-navigation/index.tsx:73]: lol, sorry. thanks for catching it

File: packages/components/src/components/hive-navigation/index.tsx (hunk 2 above)
Comment (hasparus) [reply packages/components/src/components/hive-navigation/index.tsx:80]: good idea

**Review by saihaj — APPROVED** [review]

**Pull request merged**